README.md

```diff
@@ -113,6 +120,32 @@ To contribute:
 
 ### Development
 
+- Clone repository
+
+  - Generate your SSH keys as suggested [here](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
```

Comment:
If we're really recommending this kind of thing (which I think is beyond the scope of this project) then we could recommend the GitHub CLI instead as it is simpler https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account?tool=cli